packages/lib/server/getServerErrorFromUnknown.ts

```diff
@@ -52,9 +52,11 @@ export function getServerErrorFromUnknown(cause: unknown): HttpError {
     return getHttpError({ statusCode: 400, cause });
   }
   if (cause instanceof HttpError) {
-    const redactedCause = redactError(cause);
+    const _ = redactError(cause);
     return {
-      ...redactedCause,
+      name: cause.name,
+      message: cause.message ?? "",
+      stack: cause.stack,
```

Comment:
The `redactError` method was intentionally introduced to avoid a situation where the Prisma connection information was being passed to the client. Are we sure these changes won't reintroduce that?

Reply:
I don't think so, here the function is just there to do some logging and is still being used. it's returning an error which we  used to spread with the spread operator. but js Errors don't have any enumerable properties so spread has no effect

Reply:
added redactedCuse back and removed stack just in case

Reply:
ok cool. @hariombalhara can you please take a look here as you had originally introduced this?